core/scss/utilities/mixins/global-footer/index.scss

```diff
@@ -0,0 +1,112 @@
+@charset "UTF-8";
+
+//
+// Global Footer
+//
+// Mixins for the global footer.
+//
+// Style guide: Mixins.Organisms.Footer
+//
+@mixin global-footer {
+  @include centered-grid-container;
+  background-color: $color-cardinal-red;
+  color: $color-white;
+  padding: 2.5rem 0;
+
+  .global-footer__container {
+    @include display-grid;
+
+    @media #{$breakpoint-sm} {
+      justify-items: left;
+      @include grid-columns(1fr 11fr); // Can't use repeat() function in IE
+      padding-left: $gutter-sm;
+      padding-right: $gutter-sm;
+      @include grid-column-gap($gutter-sm);
+    }
+
+    @media #{$breakpoint-md} {
+      justify-items: left;
+      @include grid-columns(1fr 11fr); // Can't use repeat() function in IE
+      padding-left: $gutter-md;
+      padding-right: $gutter-md;
+      @include grid-column-gap($gutter-md);
+    }
+
+    @media #{$breakpoint-lg} {
+      justify-items: left;
+      @include grid-columns(1fr 11fr); // Can't use repeat() function in IE
+      padding-left: $gutter-lg;
+      padding-right: $gutter-lg;
+      @include grid-column-gap($gutter-lg);
+    }
+
+    @media #{$breakpoint-xl} {
+      justify-items: left;
+      @include grid-columns(1fr 11fr); // Can't use repeat() function in IE
+      padding-left: $gutter-xl;
+      padding-right: $gutter-xl;
+      @include grid-column-gap($gutter-xl);
+    }
+
+    grid-template-columns: 100%;
+    grid-template-rows: auto auto;
+    justify-items: center;
+    padding-left: $gutter-xs;
+    padding-right: $gutter-xs;
+  }
+
+  .global-footer__brand {
+    a {
+      display: block;
+    }
+  }
+
+  .global-footer__nav {
+    @media #{$breakpoint-xs-only} {
+      @include display-grid;
+      @include grid-columns(1fr 1fr);
+      justify-content: center;
+    }
+  }
+
+  .global-footer__copyright {
+    font-size: 1.5rem;
+    font-weight: 300;
+  }
+
+  .global-footer__menu {
```

Comment:
Some of these styles could be better refactored into a mixin for a vertical menu.